service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -373,7 +372,7 @@ public static LagomClientFactory create(String serviceName, ClassLoader classLoa
 
     // Akka
     ActorSystem actorSystem = ActorSystem.create("lagom-client", configuration, classLoader);
-    Materializer materializer = ActorMaterializer.create(actorSystem);
+    Materializer materializer = Materializer.matFromSystem(actorSystem);
```

Comment:
`matFromSystem` is an `implicit def` and the intended purpose wasn't to use it like this. I'd prefer
```
SystemMaterializer.get(actorSystem).materializer
```

or from Scala
```
SystemMaterializer(actorSystem).materializer
```

Reply:
Actually, the original intention was to be able run streams with just an ActorSystem, instead of creating a separate Materializer. Look at how the `materializer` is used here, and maybe it's not needed at all and you can use
```
stream.run(system)
```